documentation-site/examples/use-styletron/overrides.js

```diff
@@ -12,14 +12,16 @@ export default () => {
         BaseButton: props => {
           return (
             <button
-              {...props}
```

Comment:
I think we still need to spread the props here to pass handlers if any.

Reply:
we cannot spread them, as they are invalid on the `button` component - as it serves more of an example, I think it should be up for the consumer to do that